src/components/Header.vue

```diff
@@ -127,8 +136,11 @@ export default {
   methods: {
     async commit() {
       await axios.put('/api/commit')
-
       this.$refs.commit.show()
+    },
+    async logout() {
```

Comment:
```suggestion
    logout() {
```

It's probably a smart idea to tell the backend to invalidate this token too.

```
axios.put('/api/logout', localStorage.getItem('auth:token'))
```